ftl-build/arm/Dockerfile

```diff
@@ -16,17 +16,25 @@ RUN mkdir rpi_tools && cd rpi_tools && git init && git remote add origin https:/
 RUN wget ftl.pi-hole.net/libraries/libgmp.a -O /usr/local/lib/libgmp.a && \
     wget ftl.pi-hole.net/libraries/libnettle.a -O /usr/local/lib/libnettle.a && \
     wget ftl.pi-hole.net/libraries/libhogweed.a -O /usr/local/lib/libhogweed.a && \
-    wget ftl.pi-hole.net/libraries/libcap.so.2.25 -O /usr/local/lib/libcap.so
+    wget ftl.pi-hole.net/libraries/libcap.so.2.25 -O /usr/local/lib/libcap.so && \
+    wget ftl.pi-hole.net/libraries/libidn2.so.0.1.4 -O  /usr/local/lib/libidn2.so
```

Comment:
Do we still need to download the libidn2?  libidn2-0-dev:armel exists. https://packages.debian.org/stretch/armel/libidn2-0-dev/download

Reply:
It tried it by changing
```
CC="arm-linux-gnueabihf-gcc -I/usr/include -I/usr/include/arm-linux-gnueabi -L/usr/lib/arm-linux-gnueabi"
```
but the final linking step fails:
```
/usr/bin/../lib/gcc/arm-linux-gnueabihf/4.9.3/../../../../arm-linux-gnueabihf/bin/ld: error: pihole-FTL uses VFP register arguments, /usr/lib/arm-linux-gnueabi/libidn.a(rfc3454.o) does not
/usr/bin/../lib/gcc/arm-linux-gnueabihf/4.9.3/../../../../arm-linux-gnueabihf/bin/ld: failed to merge target specific data of file /usr/lib/arm-linux-gnueabi/libidn.a(rfc3454.o)
```
(and a ton more of similar ones)

I will again look at cross-compiling `libidn` from scratch